vignettes/estimate_ascertainment.Rmd

```diff
@@ -0,0 +1,279 @@
+---
+title: "Estimating the proportion of cases that are ascertained during an outbreak"
+output:
+  bookdown::html_vignette2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: resources/library.json
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Estimating the proportion of cases that are ascertained during an outbreak}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  dpi = 300,
+  fig.width = 5, fig.height = 3
+)
+```
+
+Cases, infections or hospitalisations and deaths might be under-ascertained during an outbreak of an infectious disease for reasons including testing capacity, criteria not being sufficient to identify all infections, or symptom-based testing rather than random sampling, or changes in case definition over time.
```

Comment:
Thanks for spotting this, again I'm not sure why this was picked - what would you suggest as the reference value for this time period in this context?

Reply:
Sorry, I don't understand your question. I think it would be good to perhaps add a sentence below in the example (when citing verity) to clarify what is meant by a case and perhaps add a caveat that we're referring to detection of clinical cases and their relationship to reported deaths.

Reply:
I mean what's a good reference value for the 'true' CFR for the early phase of the pandemic in the UK? Can add caveat too.

Reply:
> I mean what's a good reference value for the 'true' CFR for the early phase of the pandemic in the UK?

I genuinely don't think there is one.